packages/shared/lib/core/token/utils/isValidErc20.ts

```diff
@@ -0,0 +1,6 @@
+import type { TokenMetadata } from '@core/wallet'
+import { MAX_SUPPORTED_ERC20_DECIMALS } from '@core/wallet/constants'
+
+export function isValidErc20(tokenMetadata: TokenMetadata): boolean {
```

Comment:
Feels strange to not have token appended to the end of this function name, i.e. `isValidErc20Token`